### PR TITLE
fix: write failed when value is zero

### PR DESCRIPTION
**Pull request by daviderli614**
## What's changed and what's your intention?

When the id value is 0, the write result is as follows, the id column shows null, we should allow 0 value to be written.
```
mysql> select *from monitors_with_tag;
+------+-----------+--------+------+-------------+---------+----------------------------+
| id   | host      | memory | cpu  | temperature | running | ts                         |
+------+-----------+--------+------+-------------+---------+----------------------------+
| NULL | 127.0.0.1 |      1 |  1.1 |          -1 |       1 | 2024-08-14 09:09:29.520000 |
| NULL | 127.0.0.1 |      1 |  1.1 |          -1 |       1 | 2024-08-14 09:09:29.710000 |
| NULL | 127.0.0.2 |      2 |    2 |          -2 |       1 | 2024-08-14 09:09:29.520000 |
| NULL | 127.0.0.2 |      2 |    2 |          -2 |       1 | 2024-08-14 09:09:29.710000 |
+------+-----------+--------+------+-------------+---------+----------------------------+
```

## Checklist

- [ ]  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)

### CODE REVIEW

File: examples/object/README.md [hunk 1]
```diff
@@ -24,14 +24,16 @@ Your can using [MySQL Client](https://docs.greptime.com/user-guide/clients/mysql
 ```shell
 $ mysql -h 127.0.0.1 -P 4002 public
 
-mysql> select * from monitors_with_tag;
+mysql> select *from monitors_with_tag;
```
Comment (tisonkun) [inline examples/object/README.md:27]:
```suggestion
mysql> select * from monitors_with_tag;
```

**Comment by daviderli614** [comment]
PTAL @yuanbohan @fengjiachun

File: request/request.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func (r *Request) WithTables(tables ...*table.Table) *Request {
 }
 
 func (r *Request) IsZero() bool {
-	return r.tables == nil || len(r.tables) == 0
+	return r.tables == nil
```
Comment (zyy17) [inline request/request.go:55]: suggestion: is it better to use `IsNilTable()` or `IsTableSet()`?

Comment (daviderli614) [reply request/request.go:55]: I modified it.

File: schema/field.go [hunk 3]
```diff
@@ -180,7 +180,7 @@ func parseIntOrTimeValue(typ gpb.ColumnDataType, val reflect.Value) (*gpb.Value,
 
 func parseValue(typ gpb.ColumnDataType, val reflect.Value) (*gpb.Value, error) {
 	val = reflect.Indirect(val)
-	if !val.IsValid() || val.IsZero() {
+	if !val.IsValid() {
```
Comment (yuanbohan) [inline schema/field.go:183]: 👍

**Review by yuanbohan — APPROVED** [review]
LGTM

**Pull request merged**